iCubTelemVizServer/devices/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2021 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the Apache-2.0 license
+
+cmake_minimum_required(VERSION 3.5)
+project(matlab-whole-body-simulator)
+
+# There is nothing to build, only the fake battery config files need to be copied to the install folder:
+
+set(YARP_ROBOTS_INSTALL_DIR "share/yarp/robots" CACHE
+    STRING "Location (relative to the install prefix) where the robot configuration files are installed.")
+
+set(ICUB_MODELS_TO_INSTALL iCubGazeboV2_5)
+
+set(INSTALL_FAKE_BATTERY_DEVICE_CONFIG_FILES OFF CACHE
+    BOOL "Install the fake battery configuration files required for emulating the battery status data stream.")
+
+if (${INSTALL_FAKE_BATTERY_DEVICE_CONFIG_FILES})
+    foreach(YARP_ROBOT_NAME ${ICUB_MODELS_TO_INSTALL})
```

Comment:
Nit. This initially confused me since ``YARP_ROBOT_NAME`` is already used in other contexts.

Reply:
Changed to `robotName`.